### PR TITLE
[GEOS-10638] drop Java 8 support from docs

### PR DESCRIPTION
**Pull request by euanmitchell**
[GEOS-10638](https://osgeo-org.atlassian.net/browse/GEOS-10638)

  

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md).
- [ ] I have sent a [Contribution Licence Agreement](https://docs.geoserver.org/latest/en/developer/policies/committing.html) (not required for small changes, e.g., fixing typos in documentation).
- [x] First PR targets the `main` branch (backports managed later; ignore for branch specific issues).
- [ ] All the build checks are green ([see automated QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html)).

For core and extension modules:

- [ ] New unit tests have been added covering the changes.
- [x] [Documentation](https://github.com/geoserver/geoserver/tree/main/doc/en/user/source) has been updated (if change is visible to end users).
- [ ] The [REST API docs](https://github.com/geoserver/geoserver/tree/main/doc/en/api/1.0.0) have been updated (when changing configuration objects or the REST controllers).
- [x] There is an issue in the [GeoServer Jira](https://osgeo-org.atlassian.net/browse/GEOS/summary) (except for changes that do not affect administrators or end users in any way).
- [x] Commit message(s) must be in the form ``[GEOS-XYZWV] Title of the Jira ticket``.
- [ ] Bug fixes and small new features are presented as a single commit.
- [ ] Each commit has a single objective (if there are multiple commits, each has a separate JIRA ticket describing its goal).

### CODE REVIEW

**Comment by jodygarnett** [comment]
Code sprint activity; documenation rather than functional change so I am not going to require a contribution agreement.

File: doc/en/user/source/configuration/status.rst [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 ======
-The Server Status page has two tabs to summarize the current status of GeoServer. The Status tab provides a summary of server configuration parameters and run-time status. The modules tab provides the status of the various modules installed on the server. This page provides a useful diagnostic tool in a testing environment. 
+The Server Status page has two tabs to summarize the current status of GeoServer. The Status tab provides a summary of server configuration parameters and run-time status. The modules tab provides the status of the various modules installed on the server. This page provides a useful diagnostic tool in a testing environment.
 
 Server Status
 -------------
 
 .. figure:: img/server_status.png
-   
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:11]: what space change due to editor

**Review by jodygarnett — CHANGES_REQUESTED** [review]
A small number of whitepace changes due to text editor

File: doc/en/user/source/configuration/status.rst (hunk 1 above)
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:11]:
```suggestion
   
```

File: doc/en/user/source/configuration/status.rst [hunk 2]
```diff
@@ -26,14 +26,14 @@ The following table describes the current status indicators.
      - Shows the path to the GeoServer data directory (``GEOSERVER_DATA_DIR`` property).
    * - Locks
      - A WFS has the ability to lock features to prevent more than one person from updating the feature at one time.  If data is locked, edits can be performed by a single WFS editor. When the edits are posted, the locks are released and features can be edited by other WFS editors. A zero in the locks field means all locks are released. If locks is non-zero then a client has reserved some content for editing.
-       
-       Press :guilabel:`Free locks` to release all feature locks currently held by the server and update the field value to zero. 
-       
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:29]:
```suggestion
      
      Press :guilabel:`Free locks` to release all feature locks currently held by the server and update the field value to zero.
      
```

File: doc/en/user/source/configuration/status.rst [hunk 3]
```diff
@@ -26,14 +26,14 @@ The following table describes the current status indicators.
...
+       Press :guilabel:`Free locks` to release all feature locks currently held by the server and update the field value to zero.
+
    * - Connections
      - Refers to the numbers of stores GeoServer is connected to, in the above case 9, that are enabled and available.
    * - Memory Usage
      - The amount of memory currently used by GeoServer. In the above example, 174 MB of memory used out of a total of 3.56 GB available to Java.
-       
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:36]:
```suggestion
      
```

File: doc/en/user/source/configuration/status.rst [hunk 4]
```diff
@@ -42,20 +42,20 @@ The following table describes the current status indicators.
    * - Available Fonts
      - Shows the number of fonts available. Selecting the link will show the full list.
    * - Native JAI
-     - GeoServer uses `Java Advanced Imaging <https://www.oracle.com/java/technologies/java-archive-downloads-java-client-downloads.html>`__ (JAI) framework for image rendering and coverage manipulation. When installed (true) native operations are available for image processing in a Java 8 environment.
-     
-       We recommend the use of JAI-EXT operations for greater stability. 
+     - GeoServer uses `Java Advanced Imaging <https://www.oracle.com/java/technologies/java-archive-downloads-java-client-downloads.html>`__ (JAI) framework for image rendering and coverage manipulation.
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:46]:
```suggestion
      
```

File: doc/en/user/source/configuration/status.rst [hunk 5]
```diff
@@ -42,20 +42,20 @@ The following table describes the current status indicators.
...
+       We recommend the use of JAI-EXT operations for greater stability.
 
    * - Native JAI ImageIO
-     - GeoServer uses `JAI Image IO <https://docs.oracle.com/javase/6/docs/technotes/guides/imageio/>`__ (JAI) framework for raster data loading and image encoding. When installed (true) native encoders are available for WCS and WMS output in a Java 8 environment.
-     
+     - GeoServer uses `JAI Image IO <https://docs.oracle.com/javase/6/docs/technotes/guides/imageio/>`__ (JAI) framework for raster data loading and image encoding.
+
        We recommend use of libjpeg-turbo for those interested in increasing encoding performance.
-       
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:53]:
```suggestion
      
```

File: doc/en/user/source/configuration/status.rst [hunk 6]
```diff
@@ -73,38 +73,38 @@ The following table describes the current status indicators.
...
    * - Resource cache
      - GeoServer maintains a resource cache with connections to stores, feature type definitions, external graphics, font definitions, and CRS definitions. This includes custom CRS definitions defined in data directory :file:`user_projections/epsg.properties`.
-     
-       Press :guilabel:`Clear` to empty the resource cache. This will force GeoServer to reconnect to stores, reconnect to databases, re-read icon and font information, and reload custom CRS definitions. 
+
+       Press :guilabel:`Clear` to empty the resource cache. This will force GeoServer to reconnect to stores, reconnect to databases, re-read icon and font information, and reload custom CRS definitions.
    * - Configuration and catalog
      - GeoServer keeps its configuration data in memory.
-       
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:80]:
```suggestion
      
```

File: doc/en/user/source/configuration/status.rst [hunk 7]
```diff
@@ -73,38 +73,38 @@ The following table describes the current status indicators.
      - Refers to the number of times, e.g. 157, the server configuration has been modified.
    * - Resource cache
      - GeoServer maintains a resource cache with connections to stores, feature type definitions, external graphics, font definitions, and CRS definitions. This includes custom CRS definitions defined in data directory :file:`user_projections/epsg.properties`.
-     
-       Press :guilabel:`Clear` to empty the resource cache. This will force GeoServer to reconnect to stores, reconnect to databases, re-read icon and font information, and reload custom CRS definitions. 
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:76]:
```suggestion
      
```

File: doc/en/user/source/configuration/status.rst [hunk 8]
```diff
@@ -73,38 +73,38 @@ The following table describes the current status indicators.
...
 
 Module Status
 -------------
 
-The :guilabel:`Modules` tab provides a summary of the status of all installed modules in the running server. 
+The :guilabel:`Modules` tab provides a summary of the status of all installed modules in the running server.
 
 .. figure:: img/module_status.png
-   
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:91]:
```suggestion
   
```

File: doc/en/user/source/configuration/status.rst [hunk 9]
```diff
@@ -73,38 +73,38 @@ The following table describes the current status indicators.
...
 Field Descriptions
 ^^^^^^^^^^^^^^^^^^
 
 .. list-table::
    :widths: 20 80
    :header-rows: 1
-   
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:100]:
```suggestion
   
```

File: doc/en/user/source/configuration/status.rst [hunk 10]
```diff
@@ -73,38 +73,38 @@ The following table describes the current status indicators.
...
    * - Module Name
      - Human readable name of the module, this links to a popup containing the full details and messages of the module
    * - Module ID
      - The internal package name of the module
    * - Available
      - Whether the module is available to GeoServer.
-       
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:107]:
```suggestion
      
```

File: doc/en/user/source/configuration/status.rst [hunk 11]
```diff
@@ -117,18 +117,18 @@ Field Descriptions
 
 .. figure:: img/module_popup.png
    :scale: 50%
-   
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:120]:
```suggestion
   
```

File: doc/en/user/source/configuration/status.rst [hunk 12]
```diff
@@ -117,18 +117,18 @@ Field Descriptions
...
 
 System Status
 -------------
 
-The :guilabel:`System Status` tab provides extra information about the system environment GeoServer is running in.  This provides an overview of the status of the GeoServer instance. 
+The :guilabel:`System Status` tab provides extra information about the system environment GeoServer is running in.  This provides an overview of the status of the GeoServer instance.
 
 .. figure:: img/system_status.png
-   
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:131]:
```suggestion
   
```

File: doc/en/user/source/configuration/status.rst [hunk 13]
```diff
@@ -246,16 +246,16 @@ For information on the live Java Runtime Environment the :guilabel:`JVM Console`
 Press :guilabel:`Thread Dump` for a summary of all active threads. This is primarily used to troubleshoot performance issues and a non-responsive system. This can be used to identify when significant work is happening in the background, or if threads are stuck waiting on a resource.
 
 .. figure:: img/thread_dump.png
-   
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:249]:
```suggestion
   
```

File: doc/en/user/source/configuration/status.rst [hunk 14]
```diff
@@ -246,16 +246,16 @@ For information on the live Java Runtime Environment the :guilabel:`JVM Console`
...
    Thread Dump console output
-   
-   
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:251]:
```suggestion
   
   
```

File: doc/en/user/source/configuration/status.rst [hunk 15]
```diff
@@ -246,16 +246,16 @@ For information on the live Java Runtime Environment the :guilabel:`JVM Console`
...
+
 Press :guilabel:`Heap Dump` for an overview of memory use. This can be used to troubleshoot systems that are encountering a memory leak over time.
 
 .. figure:: img/heap_dump.png
-   
+
    Heap Dump console output
-   
+
```
Comment (jodygarnett) [inline doc/en/user/source/configuration/status.rst:258]:
```suggestion
   
```

File: doc/en/user/source/installation/index.rst [hunk 16]
```diff
@@ -7,18 +7,16 @@ There are many ways to install GeoServer on your system. This section will discu
 
 .. note:: To run GeoServer as part of an existing servlet container such as Tomcat, please see the :ref:`installation_war` section.
 
-.. warning:: GeoServer requires a Java 8 or Java 11 environment (JRE) to be installed on your system, available from `OpenJDK <http://openjdk.java.net>`__, `AdoptOpenJDK <https://adoptopenjdk.net>`__ for Windows and macOS installers, or provided by your OS distribution.
-   
+.. warning:: GeoServer requires a Java 11 or Java 17 environment (JRE) to be installed on your system, available from `OpenJDK <http://openjdk.java.net>`__, `Adoptium <https://adoptium.net>`__ for Windows and macOS installers, or provided by your OS distribution.
+
```
Comment (jodygarnett) [inline doc/en/user/source/installation/index.rst:11]:
```suggestion
   
```

File: doc/en/user/source/installation/linux.rst [hunk 17]
```diff
@@ -27,7 +27,7 @@ Installation
 #. Add an environment variable to save the location of GeoServer by typing the following command:
 
     .. code-block:: bash
-    
+
```
Comment (jodygarnett) [inline doc/en/user/source/installation/linux.rst:30]:
```suggestion
   
```

File: doc/en/user/source/installation/linux.rst [hunk 18]
```diff
@@ -40,7 +40,7 @@ Installation
 #. Start GeoServer by changing into the directory ``geoserver/bin`` and executing the ``startup.sh`` script:
 
     .. code-block:: bash
-       
+
```
Comment (jodygarnett) [inline doc/en/user/source/installation/linux.rst:43]:
```suggestion
      
```

File: doc/en/user/source/installation/war.rst [hunk 19]
```diff
@@ -52,7 +52,7 @@ Add to ``<Host name=...`` section this new ErrorReportValve entry:
  ...
       <Host name="localhost"  appBase="webapps"
             unpackWARs="true" autoDeploy="true">
-		
+
```
Comment (jodygarnett) [inline doc/en/user/source/installation/war.rst:55]:
```suggestion
		
```

File: doc/en/user/source/installation/win_binary.rst [hunk 20]
```diff
@@ -53,7 +53,7 @@ Running
    If you see the GeoServer Welcome page, then GeoServer is successfully installed.
 
    .. figure:: images/success.png
-      
+
```
Comment (jodygarnett) [inline doc/en/user/source/installation/win_binary.rst:56]:
```suggestion
      
```

**Review by jodygarnett — APPROVED** [review]

**Comment by jodygarnett** [comment]
Cleaned up the whitespace changes and removed reference to java 8 in developers guide, looks good to me.

**Comment by ianturton** [comment]
Lgtm

**Comment by jodygarnett** [comment]
The conflict is a new section at the end of the page, not sure why it could not figure it out.

**Comment by jodygarnett** [comment]
With the proposal accepted, and the RC out this can now be merged to main

**Pull request merged**